client/shaders/nodes_shader/opengl_fragment.glsl

```diff
@@ -56,10 +57,9 @@ vec4 applyToneMapping(vec4 color)
 void main(void)
 {
 	vec3 color;
-	vec2 uv = gl_TexCoord[0].st;
+	vec2 uv = varTexCoord.st;
 
 	vec4 base = texture2D(baseTexture, uv).rgba;
-
 #ifdef USE_DISCARD
 	// If alpha is zero, we can just discard the pixel. This fixes transparency
 	// on GPUs like GC7000L, where GL_ALPHA_TEST is not implemented in mesa.
```

Comment:
Should probably be a comment here that says that discard is needed for normal GLES2 too.

Reply:
Okay.